requirements/edx/custom.txt

```diff
@@ -1,6 +1,7 @@
 # Custom requirements to be customized by individual OpenEdX instances
 
 # When updating a hash of an XBlock that uses xblock-utils, please update its version hash in github.txt.
+-e git+https://github.com/edx/xblock-utils.git@08e5a5a9fc8ab46b627435427fd7e04c20809009#egg=xblock-utils
```

Comment:
@e-kolpakov It's really confusing to have the same requirement in two different places - I can see this confusing other people in the future, and creating other issues.

You've mentioned that the issue the build fails is that https://github.com/edx-solutions/xblock-mentoring/blob/master/setup.py#L73 requires xblock-utils without specifying where it is. Would it solve the issue to remove the reference to xblock-utils there? Since the dependency would already be in the requirements, it's also already a duplicate? https://github.com/edx-solutions/xblock-mentoring/blob/master/requirements.txt